package.json

```diff
@@ -15,10 +15,22 @@
     "test:system": "./bin/system-tests.sh",
     "test:unit": "ava tests/unit/*.js",
     "typecheck": "tsc",
+    "audit-prod": "npm audit --omit dev",
     "deploy": "sls deploy",
     "serve": "STAC_API_URL=http://localhost:3000 nodemon ./src/lambdas/api/local.js",
     "build-api-docs": "npm run widdershins --search false --language_tabs 'nodejs:NodeJS' 'python:Python' --summary ./packages/api-lib/api-spec.yaml -o ./docs/api.md & npm run shins --inline --logo ./docs/images/logo.png -o ./docs/index.html ./docs/api.md"
   },
+  "pre-commit": {
```

Comment:
these now run before commits. add `--no-verify` if doing local commits that shouldn't have these run.